src/main/external-resources/renderers/Samsung-9series.conf

```diff
@@ -33,8 +33,8 @@ TranscodeAudio = WAV
 #MaxVideoWidth = 3840
 #MaxVideoHeight = 2160
 SupportedVideoBitDepths = 8,10
-H264Level41Limited = true
-#H265Level41Limited = true
+H264LevelLimit = 4.1
```

Comment:
@Sami32 I expected you to fix those details, I just replaced it with a setting correspondig to the old one.